src/query/config/src/inner.rs

```diff
@@ -43,6 +43,9 @@ pub struct Config {
     // Query engine config.
     pub query: QueryConfig,
 
+    // Settings config.
+    pub settings: SettingsConfig,
```

Comment:
No need new config section. Putting the `timezone` in the `QueryConfig` section is ok.

Reply:
Ok, I see the discussion(https://github.com/datafuselabs/databend/issues/9173#issuecomment-1344018671), that's make sense.